File: 130.13.cs
using System;
using System.Drawing;
using System.Windows.Forms;

public class p130_13 : Form
{
  private TextBox txtNumA;
  private TextBox txtNumB;
  private Label lblSum;
  private Label lblDif;
  private Label lblMul;
  public static void Main()
  {
    Application.Run(new p130_13());
  }
  public p130_13()
  {
    InitializeComponent();
  }
  private void InitializeComponent()
  {
    this.ClientSize = new Size(160, 180);
    this.FormBorderStyle = FormBorderStyle.FixedSingle;
    this.MaximizeBox = false;
    
    txtNumA = new TextBox();
    txtNumA.Location = new Point(20, 20);
    txtNumA.Width = 120;
    txtNumA.TextAlign = HorizontalAlignment.Center;
    txtNumA.TextChanged += new EventHandler(this.Number_Entered);
    this.Controls.Add(txtNumA);

    txtNumB = new TextBox();
    txtNumB.Location = new Point(20, 50);
    txtNumB.Width = 120;
    txtNumB.TextAlign = HorizontalAlignment.Center;
    txtNumB.TextChanged += new EventHandler(this.Number_Entered);
    this.Controls.Add(txtNumB);

    lblSum = new Label();
    lblSum.Location = new Point(20, 80);
    lblSum.Size = new Size(120, 30);
    lblSum.TextAlign = ContentAlignment.MiddleCenter;
    lblSum.Text = "Enter";
    this.Controls.Add(lblSum);
    
    lblDif = new Label();
    lblDif.Location = new Point(20, 110);
    lblDif.Size = new Size(120, 30);
    lblDif.TextAlign = ContentAlignment.MiddleCenter;
    lblDif.Text = "Your";
    this.Controls.Add(lblDif);
    
    lblMul = new Label();
    lblMul.Location = new Point(20, 140);
    lblMul.Size = new Size(120, 30);
    lblMul.TextAlign = ContentAlignment.MiddleCenter;
    lblMul.Text = "Numbers";
    this.Controls.Add(lblMul);
  }
  protected void Number_Entered(object sender, EventArgs args)
  {
    decimal NumA, NumB;
    try
    {
      NumA = Convert.ToDecimal(txtNumA.Text);
      NumB = Convert.ToDecimal(txtNumB.Text);
    }
    catch
    {
      lblSum.Text = "Enter";
      lblDif.Text = "True";
      lblMul.Text = "Numbers!";
      return;
    }
    lblSum.Text = "+: " + Convert.ToString(NumA + NumB);
    lblDif.Text = "-: " + Convert.ToString(NumA - NumB);
    lblMul.Text = "*: " + Convert.ToString(NumA * NumB);
  }
}

File: 130.13.py
#!/usr/bin/python
import sys
from PyQt4 import QtCore, QtGui

class p130_13:
  
  def InitializeComponents(self):
    self.appApplication = QtGui.QApplication(sys.argv)
    self.wgtMain = QtGui.QWidget()
    self.grdMain = QtGui.QGridLayout()
    self.txtNumA = QtGui.QLineEdit()
    self.txtNumB = QtGui.QLineEdit()
    self.lblSum = QtGui.QLabel('Enter')
    self.lblDif = QtGui.QLabel('Your')
    self.lblMul = QtGui.QLabel('Numbers')
    
    self.txtNumA.setAlignment(QtCore.Qt.AlignCenter)
    self.txtNumA.textChanged.connect(self.Text_Changed)
    self.grdMain.addWidget(self.txtNumA, 1, 0)
    
    self.txtNumB.setAlignment(QtCore.Qt.AlignCenter)
    self.txtNumB.textChanged.connect(self.Text_Changed)
    self.grdMain.addWidget(self.txtNumB, 2, 0)
    
    self.lblSum.setAlignment(QtCore.Qt.AlignCenter)
    self.grdMain.addWidget(self.lblSum, 3, 0)
    
    self.lblDif.setAlignment(QtCore.Qt.AlignCenter)
    self.grdMain.addWidget(self.lblDif, 4, 0)
    
    self.lblMul.setAlignment(QtCore.Qt.AlignCenter)
    self.grdMain.addWidget(self.lblMul, 5, 0)
    
    self.wgtMain.setWindowTitle(' ')
    self.grdMain.setSpacing(10)
    self.wgtMain.setLayout(self.grdMain)
    self.wgtMain.setFixedSize(160, 180)

  def Text_Changed(self):
    try:
      NumA = float(str(self.txtNumA.text()))
      NumB = float(str(self.txtNumB.text()))
    except:
      self.lblSum.setText('Enter')
      self.lblDif.setText('True')
      self.lblMul.setText('Numbers!')
      return
    self.lblSum.setText('+: ' + str( NumA + NumB ))
    self.lblDif.setText('-: ' + str( NumA - NumB ))
    self.lblMul.setText('*: ' + str( NumA * NumB ))
  def __init__(self):
    self.InitializeComponents()
  
  def Run(self):
    self.wgtMain.show()
    self.appApplication.exec_()

if __name__ == '__main__' :
  myprj = p130_13()
  myprj.Run()
